instrument/IN6_Parameters.xml

```diff
@@ -17,7 +17,36 @@
 			See http://muparser.sourceforge.net/mup_features.html#idDef2 for available 
 			operators -->
 		<parameter name="formula_eff" type="string">
-			<value val="exp(-0.0565/sqrt(e))*(1.0-exp(-3.284/sqrt(e)))" />
+			<!-- The xml escape sequence &lt; below means 'less than' -->
```

Comment:
These changes will go out to users immediately once merged, not with the next release.  The look safe to me, but you might want to double check.